src/main/java/Duke.java

```diff
@@ -1,10 +1,146 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static boolean isNumeric(String command){
+        if(command == null){
+            return false;
+        }
+        try{
+            double d = Double.parseDouble(command);
+        }catch(NumberFormatException nfe) {
+            return false;
+        }
+        return true;
+    }
+    /*
+    function to check whether is numeric
+     */
+
+    public static String checkType(String command) {
+
+        String type = "invalid command";
+
+        if (command.length() >= 10) {
+            if (command.startsWith("deadline")) {
+                type = "[D]";
+                return type;
+            }
+        }
+        if (command.length() >= 7) {
+            if (command.startsWith("event")) {
+                type = "[E]";
+                return type;
+            }
+        }
+        if (command.length() >= 6){
+            if (command.startsWith("todo")) {
+                type = "[T]";
+                return type;
+            }
+            if (command.startsWith("done") && isNumeric(command.substring(5, command.length()))){
+                type = "done command";
+            }
+        }
+
+        return type;
+    }
+    /*
+    function to check the command type
+     */
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("Hello, what can I do for you?");
+
+        task[] tasks = new task[100];
+        /*
+            array to store the tasks
+         */
+        Integer NumOfTasks=0;
+        /*
+            number of tasks currently in the list
+         */
+
+        while(true){
+            String command;
+            Scanner in = new Scanner(System.in);
+            command = in.nextLine();
+            Integer length = command.length();
+            String type ="?";
+            /*
+            command is what the user types in
+            type is the type of task, event/to do/deadlines
+             */
+
+            if(command.equals("bye")){
```

Comment:
This if-else block is a bit long. Refactoring it into extracted methods would make the code clearer!